{"Modified":"2023-05-03T12:15:00","Published":"2022-01-11T13:15:00","access":{"authentication":"NONE","complexity":"LOW","vector":"LOCAL"},"assigner":"cve@mitre.org","capec":[],"cvss":2.1,"cvss-time":"2023-05-03T12:15:00","cvss-vector":"AV:L/AC:L/Au:N/C:N/I:N/A:P","cwe":"CWE-843","id":"CVE-2021-44647","impact":{"availability":"PARTIAL","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2023-05-03T12:15:00","references":["http://lua-users.org/lists/lua-l/2021-11/msg00204.html","http://lua-users.org/lists/lua-l/2021-11/msg00195.html","https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/P3EMGAQ5Y6GXJLY4K5DUOOEQT4MZ4J4F/","https://access.redhat.com/security/cve/cve-2021-44647","https://security.gentoo.org/glsa/202305-23"],"summary":"Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service.","vulnerable_configuration":[{"id":"cpe:2.3:a:lua:lua:5.4.3:*:*:*:*:*:*:*","title":"cpe:2.3:a:lua:lua:5.4.3:*:*:*:*:*:*:*"},{"id":"cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*","title":"cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:lua:lua:5.4.3:*:*:*:*:*:*:*","cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*"]}